src/engine/controls/bpmcontrol.cpp

```diff
@@ -151,46 +155,47 @@ BpmControl::~BpmControl() {
     delete m_pTranslateBeatsLater;
```

Comment:
All manual deletes should become obsolete after switching to parented_ptr/unique_ptr respectively